src/main/java/com/zufar/onlinestore/payment/converter/PaymentMethodConverter.java

```diff
@@ -2,22 +2,27 @@
 
 import com.stripe.param.PaymentMethodCreateParams;
 import com.zufar.onlinestore.payment.dto.CreatePaymentMethodDto;
-import org.springframework.stereotype.Component;
+import org.mapstruct.Mapper;
 
-@Component
-public class PaymentMethodConverter {
+@Mapper
+public interface PaymentMethodConverter {
 
-    public PaymentMethodCreateParams toPaymentMethodParams(final CreatePaymentMethodDto createPaymentMethodDto) {
-        PaymentMethodCreateParams.CardDetails cardDetails = PaymentMethodCreateParams.CardDetails.builder()
-                .setNumber(createPaymentMethodDto.cardNumber())
-                .setExpMonth(createPaymentMethodDto.expMonth())
-                .setExpYear(createPaymentMethodDto.expYear())
-                .setCvc(createPaymentMethodDto.cvc())
-                .build();
+    default PaymentMethodCreateParams toPaymentMethodParams(final CreatePaymentMethodDto createPaymentMethodDto) {
+        if (createPaymentMethodDto == null) {
+            return null;
```

Comment:
returning `null` is dangerous
throw new custom exception here with the detailed error description here
like `InvalidCreatePaymentMethodDtoException("CreatePaymentMethodDto is null")`